src/S3/S3Client.php

```diff
@@ -712,6 +712,15 @@ public static function applyDocFilters(array $api, array $docs)
         $b64 = '<div class="alert alert-info">This value will be base64 encoded on your behalf.</div>';
         $opt = '<div class="alert alert-info">This value will be computed for you it is not supplied.</div>';
 
+        // Add a note on the CopyObject docs
+        $docs['operations']['CopyObject'] .= "<p>Additional info on response behavior: if there is"
```

Comment:
Two options here considering all four in AmbiguousSuccessParser perform this way:
-the one I did: hard code all four 
-one i considered: making the AmbiguousSuccessParser's list of operations it supports public and using a for loop on that to add to the documentation.  The only problem here is that someone may add an operation with custom (different) logic to AmbiguousSuccessParser.  If it behaved the same way though, then this would automatically add this doc to them.  Let me know if you want me to implement it this way.